.php-cs-fixer.php

```diff
@@ -11,6 +11,9 @@
     ->in([
         __DIR__ . '/src',
         __DIR__ . '/tests',
+    ])
+    ->exclude([
+        __DIR__ . 'tests/app/var',
     ]);
```

Comment:
`exclude` is a bit tricky. The PhpDoc says:

> Directories passed as argument must be relative to the ones defined with the in() method.

which means it's only:

```suggestion
    ->exclude([
        'app/var',
    ]);
```

because `__DIR__ . '/tests'` is already in the `in()` method.

Reply:
Does this work? Should we apply it?